docs/react-native-devtools.md

```diff
@@ -0,0 +1,99 @@
+---
+id: react-native-devtools
+title: React Native DevTools
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+React Native DevTools is our new debugging experience featuring an end-to-end rewrite of our debugger stack. It aims to be more deeply integrated and fundamentally more reliable than previous debugging methods in React Native.
+
+![React Native DevTools opened to the "Welcome" pane](/docs/assets/debugging-rndt-welcome.jpg)
+
+React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](http://localhost:3000/docs/next/debugging-native-code)).
+
+<details>
+<summary>**💡 Compatibility** — released in 0.76</summary>
+
+React Native DevTools supports all React Native 0.76 apps (and onwards) running Hermes. It replaces the previous Flipper, Experimental Debugger, and Hermes debugger (Chrome) entry points.
+
+It is not possible to set up React Native DevTools with any older versions of React Native.
+
+- **Chrome Browser DevTools — unsupported**
+  - Connecting to React Native via `chrome://inspect` is no longer supported. Features may not work correctly, as the latest versions of Chrome DevTools (which are built to match the latest browser capabilities and APIs) have not been tested, and this frontend lacks our customisations. Instead, we ship a supported version with React Native DevTools.
+- **Visual Studio Code — unsupported** (pre-existing)
+  - Third party extensions such as [Expo Tools](https://github.com/expo/vscode-expo) and [Radon IDE](https://ide.swmansion.com/) may have improved compatibility, but are not directly supported by the React team.
+
+</details>
+<details>
+<summary>**💡 Feedback & FAQs**</summary>
```

Comment:
why in a details/summary section? 

I think this is relevant and it can have its own sub section

```suggestion
## 💡 Feedback & FAQs
```

Reply:
But this contains a reference that the users might not find when they need it. I might want to go to the discussion and I might miss it when I skim through the document because it is collapsed in the `details`...

Reply:
I feel that "FAQs" is a good enough title callout for people to discover this. I'd like readers to hit the content of this page (with new sections incoming) sooner.